### ISSUE TITLE
Screen readers are not narrating role (as dialog) once the dialog is displayed on invoking 'help' control.

### ISSUE THREAD

**Issue by JamesFerebee** [microsoft/SqlNexus#342]
Impact on a Person with Disability (PWD):
Users who rely on screen readers will get misguided and not aware of exact information about the content of the dialog as Screen readers are not narrating role (as dialog) once the dialog is displayed on invoking 'help' control.

Test Environment:​
OS: Windows 11 [Version 22H2 (OS Build 22621.2283)].

Desktop application: SQL Nexus.

Tool: Narrator, JAWS (2022.2110.6).

Prerequisite:
1. Install SQL NEXUS application on the desktop.
2. Enable Narrator/JAWS.

Repro Steps:
Step 1: Open the SQL Nexus app in desktop.
Step 2: Connect to server Dialog gets displayed. Connect to the server by invoking connect button.
Step 3: Navigate to Edit Custom row set in left navigation.
Step 4: Navigate to 'Help' button and invoke it.
Step 5: Verify whether Screen readers are narrating role (as dialog) once the dialog is displayed on invoking 'help' control or not.
 
Actual Result:
Screen readers are not narrating role (as dialog) once the dialog is displayed on invoking 'help' control.
Current Observation: Narrator is announcing as 'Ok button Current version doesn't have UI to add your own column data types...'.
Observation:
Visual dialog name is also not provided for the dialog.

Expected Result:
Screen readers should narrate role (as dialog) once the dialog is displayed on invoking 'help' control.
Ex: Narrator should announce as ''Visual name' Dialog Ok button Current version doesn't have UI to add your own column data types...'.
Ensure that Visual dialog name should be provided for the dialog that describes the purpose of the dialog.

**Comment by JamesFerebee** [comment]
1711707

**Comment by PiJoCoder** [comment]
Fixed by PR https://github.com/microsoft/SqlNexus/pull/365
